### DISCUSSION TITLE
Software to design Wireframes ?

### DISCUSSION THREAD

**Discussion by claushellsing** [primefaces/community#2159] [PrimeVue] [answered]
Hi , I wonder what software is used to design the wireframe ?

**Comment by vieira-brz** [comment, marked as answer]
**Free and Web-Based:**

* **Figma:**  A powerhouse for collaborative design, offering excellent wireframing capabilities, easy sharing, and real-time collaboration. It's fantastic for teams.
* **Adobe XD:** Adobe's vector-based design tool provides a streamlined experience for wireframing, prototyping, and design. 
* **Balsamiq:**  Known for its quick and easy sketching-style wireframes. It's intuitive for beginners and focused on the core layout.
* **Whimsical:**  Great for brainstorming and collaboration, offering a wide range of diagram types, including wireframes.

**Desktop-Based:**

* **Sketch:**  A popular choice for Mac users, with a focus on design and prototyping.
* **Axure RP:**  Powerful tool for detailed wireframing, prototyping, and interaction design. 
* **Justinmind:**  Another robust option for prototyping with a focus on user interactions and complex workflows. 

**Specialized for Wireframing:**

* **MockFlow:**  An online tool specifically designed for wireframing with a wide range of templates and features. 
* **Wireframe.cc:**  Simple and intuitive, focusing on creating clear and functional wireframes.

**Choosing the Best Fit:**

* **Collaboration:**  If you need real-time collaboration, Figma is a top pick.
* **Ease of Use:** Balsamiq is fantastic for beginners or those who prefer quick sketching.
* **Prototyping:**  Adobe XD, Axure RP, and Justinmind are strong for prototyping complex user interactions. 
* **Specific Needs:** MockFlow and Wireframe.cc are excellent if you want to focus solely on wireframing. 

**Important Factors:**

* **Learning Curve:**  Some tools have steeper learning curves than others. 
* **Features:** Consider what features are essential to your workflow, like prototyping, collaboration, or specific libraries.
* **Pricing:**  Most offer free plans for individual use, with paid plans for more advanced features or teams.
